airflow/contrib/operators/docker_swarm_operator.py

```diff
@@ -127,7 +133,37 @@ def _run_image(self):
         # wait for the service to start the task
         while not self.cli.tasks(filters={'service': self.service['ID']}):
             continue
+
+        logs = self.cli.service_logs(self.service['ID'], follow=True, stdout=True, stderr=True, is_tty=self.tty)
+        line = ''
+        _stream_logs = self.enable_logging # Status of the service_logs' generator
         while True:
```

Comment:
Maybe I do something wrong but with
```
DockerSwarmOperator(
        task_id='blub',
        default_args=default_args,
        environment=dict(os.environ),
        docker_url=os.environ['DOCKER_MASTER_URL'],
        command="""python -c 'import sys; print("should kick of workflow here"); sys.exit(0)'""",
        image='python:3.7',
        auto_remove=True,
        dag=dag,
    )
```
My service just hangs indefinitely...

Reply:
Does it go away when you remove this commit? Does it go away if you set `enable_loggin=False`?

Reply:
Can you try with `command=sleep 10` maybe?

Reply:
Sadly, neither of those tests make my job finish. Quite possibly no changes in this PR are to blame for that.

Reply:
Ahh.. if removing this patch doesn't work either for you then it should be something else. Marking this as resolved but let me know if u need any help debugging the issue.